activity_browser/app/ui/panels/panel.py

```diff
@@ -91,3 +92,7 @@ def close_tab(self, index):
         widget.deleteLater()
         self.removeTab(index)
         signals.activity_tabs_changed.emit()
+
+    def close_all_activity_tabs(self):
+        for i in range(len(activity_cache)):
```

Comment:
This only deletes the first half of the open tabs. If you have two open tabs and delete the 0th tab, the first will now be the 0th and deleting the now non-existent first raises the following error:
```python
Traceback (most recent call last):
  File "/home/adrian/Git/github/AB/activity-browser/activity_browser/app/ui/panels/panel.py", line 100, in close_all_activity_tabs
    self.close_tab(i)
  File "/home/adrian/Git/github/AB/activity-browser/activity_browser/app/ui/panels/panel.py", line 92, in close_tab
    widget.deleteLater()
AttributeError: 'NoneType' object has no attribute 'deleteLater'
```
Possible fix would be to start deleting from the highest index:
```python
for i in reversed(range(len(activity_cache))):
    ...
```